nau_openedx_extensions/templates/certificate_export/certificate_export.html

```diff
@@ -0,0 +1,38 @@
+## mako
+<%page expression_filter="h"/>
+<%! from django.utils.translation import gettext as _ %>
+
+<div class="certificate-export-section">
+  <p>${_("In this section you can export the course certificates in different formats.")}</p>
+
+  <div class="certificate-export-controls">
+    <h3>${_("Export options")}</h3>
+
+    <div class="action-item">
+      <div class="action-item-content">
+        <p>${_("Download a CSV file with the details of all certificates generated for this course.")}</p>
+        <button class="btn btn-primary" id="export-csv-certificates"
+                data-endpoint="${certificate_export_url}">
+            ${_("Export certificates CSVs")}
+        </button>
+      </div>
+    </div>
+
+    <div class="action-item">
+      <div class="action-item-content">
+        <p>${_("Download a ZIP file containing all certificates in PDF format for this course.")}</p>
+        <button class="btn btn-primary" id="export-zip-certificates" data-endpoint="{{ certificate_export_pdf_url }}">
+          ${_("Export certificates PDF as ZIP")}
+        </button>
+      </div>
+    </div>
+  </div>
+
+  <div class="certificate-export-status" style="display:none;">
+    <div class="message-status error" style="display:none;">${_("Error")}</div>
+    <div class="message-status success" style="display:none;">${_("Success")}</div>
+    <div class="message-status info" style="display:none;">${_("Info")}</div>
+    <div class="message-status warning" style="display:none;">${_("Warning")}</div>
+  </div>
```

Comment:
@BryanttV @efortish Here is the only place where I think it would be beneficial to have some additional context to the user on where to get the file, if the task succeeds, because it's not clear. But we can save this for a later task,